diff --git a/next.config.mjs b/next.config.mjs
index f8dd123..0b20ad0 100644
--- a/next.config.mjs
+++ b/next.config.mjs
@@ -14,6 +14,9 @@ import { withSentryConfig } from "@sentry/nextjs";
 
 /** @type {import('next').NextConfig} */
 const config = {
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
+  output: "standalone",
   reactStrictMode: true,
   /** @param { import("webpack").Configuration } config */
   webpack(config) {
